src/rand.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Authors (alpha order): Adrian Catangiu @acatangiu, Alexandra Lordache @aghecenco, Alexandru Agache @alexandruag, Diana Popa @dianpopa, Matt Forrester @forbesmyester, Radu Weiss @raduweiss
+
```

Comment:
The module needs high level documentation which can be added with `!\\\` instead of just `\\\`.
https://doc.rust-lang.org/1.9.0/book/documentation.html#documenting-modules

Reply:
`//!`? done

Reply:
Yap.